Doc/library/asyncio-eventloop.rst

```diff
@@ -1,10 +1,15 @@
 .. currentmodule:: asyncio
 
+.. _asyncio-event-loop:
 
 ==========
 Event Loop
 ==========
 
+**Source code:** :source:`Lib/asyncio/events.py`,
+:source:`Lib/asyncio/base_events.py`
```

Comment:
In case it's not clear, both of these will appear on a single line after being rendered in Sphinx:
![image](https://user-images.githubusercontent.com/44193521/66379186-f8297e80-e982-11e9-98d2-dfca429760e8.png)

Reply:
👍